packages/components/pie-button/src/index.ts

```diff
@@ -30,15 +30,12 @@ export class PieButton extends LitElement {
 
     render () {
         const {
-            size, type, variant, disabled, isFullWidth,
+            type, disabled, isFullWidth,
+
         } = this;
 
         const classes = {
```

Comment:
is there any reason to use classMap here for a single class? We can always add it back if needed later on

Reply:
Nope – I'll edit it out